src/utils.rs

```diff
@@ -178,48 +198,68 @@ fn trim_multiline_inner(s: Cow<str>, ignore_first: bool, ch: char) -> Cow<str> {
 /// get a parent expr if any – this is useful to constrain a lint
 pub fn get_parent_expr<'c>(cx: &'c LateContext, e: &Expr) -> Option<&'c Expr> {
     let map = &cx.tcx.map;
-    let node_id : NodeId = e.id;
-    let parent_id : NodeId = map.get_parent_node(node_id);
-    if node_id == parent_id { return None; }
-    map.find(parent_id).and_then(|node|
-        if let NodeExpr(parent) = node { Some(parent) } else { None } )
+    let node_id: NodeId = e.id;
+    let parent_id: NodeId = map.get_parent_node(node_id);
+    if node_id == parent_id {
+        return None;
+    }
+    map.find(parent_id).and_then(|node| {
+        if let NodeExpr(parent) = node {
+            Some(parent)
+        } else {
+            None
+        }
+    })
 }
 
 #[cfg(not(feature="structured_logging"))]
 pub fn span_lint<T: LintContext>(cx: &T, lint: &'static Lint, sp: Span, msg: &str) {
     cx.span_lint(lint, sp, msg);
     if cx.current_level(lint) != Level::Allow {
-        cx.sess().fileline_help(sp, &format!("for further information visit \
-            https://github.com/Manishearth/rust-clippy/wiki#{}",
-            lint.name_lower()))
+        cx.sess().fileline_help(sp,
+                                &format!("for further information visit \
+                                          https://github.com/Manishearth/rust-clippy/wiki#{}",
+                                         lint.name_lower()))
     }
 }
 
 #[cfg(feature="structured_logging")]
 pub fn span_lint<T: LintContext>(cx: &T, lint: &'static Lint, sp: Span, msg: &str) {
     // lint.name / lint.desc is can give details of the lint
-    // cx.sess().codemap() has all these nice functions for line/column/snippet details
-    // http://doc.rust-lang.org/syntax/codemap/struct.CodeMap.html#method.span_to_string
+    // cx.sess().codemap() has all these nice functions for line/column/snippet
+    // details
+    // http://doc.rust-lang.org/syntax/codemap/struct.CodeMap.html#method.
+    // span_to_string
     cx.span_lint(lint, sp, msg);
     if cx.current_level(lint) != Level::Allow {
-        cx.sess().fileline_help(sp, &format!("for further information visit \
-            https://github.com/Manishearth/rust-clippy/wiki#{}",
-            lint.name_lower()))
+        cx.sess().fileline_help(sp,
+                                &format!("for further information visit \
+                                          https://github.com/Manishearth/rust-clippy/wiki#{}",
+                                         lint.name_lower()))
     }
 }
 
-pub fn span_help_and_lint<T: LintContext>(cx: &T, lint: &'static Lint, span: Span,
-        msg: &str, help: &str) {
+pub fn span_help_and_lint<T: LintContext>(cx: &T,
+                                          lint: &'static Lint,
```

Comment:
Is there a rustfmt setting to avoid these being aggressively split across lines? Same for method calls.